### PR TITLE
MPR test fix

### PR DESCRIPTION
**Pull request by labkey-sweta**
#### Rationale
MPR task test fix

#### Related Pull Requests
* 

#### Changes
*

### CODE REVIEW

File: WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java [hunk 1]
```diff
@@ -1688,7 +1688,8 @@ public void testMprDataEntry()
         waitForElement(Locator.linkWithText(PROJECT_MEMBER_ID), WAIT_FOR_JAVASCRIPT);
         _helper.setDataEntryField("title", MPR_TASK_TITLE);
         waitAndClick(Locator.name("title"));
-
+        _helper.setDataEntryField("performedby", FULL_SUBMITTER.getEmail());
+        checker().withScreenshot("Edited_title_for_MPR task");
```
Comment (labkey-jeckels) [inline WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java]: @labkey-sweta I'm not seeing this screenshot file anywhere in the artifacts. Am I looking in the wrong place?

File: WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java [hunk 2]
```diff
@@ -1682,13 +1682,14 @@ public void testMprDataEntry()
...
+        waitAndClick(Locator.name("title"));
         _extHelper.selectComboBoxItem("Assigned To:", BASIC_SUBMITTER.getGroup() + "\u00A0"); // appended with a nbsp (Alt+0160)
         _extHelper.setExtFormElementByLabel("Id:", PROJECT_MEMBER_ID + "\t");
         click(Locator.xpath("//div[./label[normalize-space()='Id:']]//input"));
         waitForElement(Locator.linkWithText(PROJECT_MEMBER_ID), WAIT_FOR_JAVASCRIPT);
-        _helper.setDataEntryField("title", MPR_TASK_TITLE);
-        waitAndClick(Locator.name("title"));
-
+        _helper.setDataEntryField("performedby", FULL_SUBMITTER.getEmail());
+        getArtifactCollector().dumpPageSnapshot("Edited_title_for_MPR task");
```
Comment (labkey-jeckels) [inline WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java]: You can safely delete this screenshot now if you like.

**Review by labkey-jeckels — APPROVED** [review]

File: WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java (hunk 1 above)
Comment (labkey-sweta) [reply WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java]: The test did not fail, so no artifacts were generated.

File: WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java (hunk 2 above)
Comment (labkey-sweta) [reply WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java]: Yup, was planning to after one successful run

**Pull request merged**